arangod/Aql/ClusterQuery.cpp

```diff
@@ -27,6 +27,7 @@
 #include "Aql/AqlTransaction.h"
 #include "Aql/ExecutionEngine.h"
 #include "Aql/Timing.h"
+#include "Aql/QueryCache.h"
```

Comment:
Some magic (don't understand how it works before), but I can't build without it (incomplete type), change only shared_ptr creattion